src/main/java/com/example/coursework2/controllers/ExamController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.coursework2.controllers;
+
+import com.example.coursework2.interfaces.ExaminerService;
+import com.example.coursework2.Question;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Set;
+
+@RestController
+public class ExamController {
+    private final ExaminerService examinerServiceImpl;
+
+    public ExamController(ExaminerService examinerServiceImpl){
+        this.examinerServiceImpl = examinerServiceImpl;
+    }
+    @GetMapping("/exam/java/question")
```

Comment:
```suggestion
    @GetMapping("/exam/java/{amount}")
```
вот такой путь должен быть (см. задание)